src/xenia/base/memory_generic.cc

```diff
@@ -43,7 +43,14 @@ void copy_and_swap_16_aligned(uint16_t* dest, const uint16_t* src,
 
 void copy_and_swap_16_unaligned(uint16_t* dest, const uint16_t* src,
                                 size_t count) {
-  for (size_t i = 0; i < count; ++i) {
+  size_t i;
+  for (i = 0; i + 8 <= count; i += 8) {
+    __m128i s = _mm_loadu_si128((__m128i*)&src[i]);
```

Comment:
Yes , here is disasm

    50:     input = _mm_loadu_si128((__m128i*)&src[i]);
00007FF63EBD7980  vmovdqu     xmm0,xmmword ptr [rcx+rax]  
    51:     output = _mm_or_si128(_mm_slli_epi16(input, 8), _mm_srli_epi16(input, 8));
00007FF63EBD7985  vpsllw      xmm1,xmm0,8  
00007FF63EBD798A  vpsrlw      xmm0,xmm0,8  
00007FF63EBD798F  vpor        xmm2,xmm1,xmm0  
00007FF63EBD7993  add         r9,8  
00007FF63EBD7997  add         r10,8  
    52:     _mm_storeu_si128((__m128i*)&dest[i], output);
00007FF63EBD799B  vmovdqu     xmmword ptr [rax],xmm2